src/utils/localFormatted/localFormatted.spec.ts

```diff
@@ -1,27 +1,19 @@
-import {formattedDate, formattedNumber} from './localFormatted';
+import {formattedNumber} from './localFormatted';
 
+const locale = 'en-US';
```

Comment:
this is a dirty hack and will fail on CI running in non-US env

Reply:
what you need to do is to `export default` curried function with the `locale` setted to `undefined` along with the named export

then the original function will be pure and you can easily test it, with the named import, and the app itself will use default export which is curried